### DISCUSSION TITLE
Adding names or words?

### DISCUSSION THREAD

**Discussion by rheaster** [Dadangdut33/Speech-Translate#73] [Q&A]
Is there a way to add names or words so they're picked up by the program?
For instance, I'm using this mostly to transcribe audio from my D&D games and some character names are out of the norm... So names like "Samiya" are transcribed as "Samir" "Samyer" etc.
"Veera" becomes "Vira" "Vera" "Vyra" etc.

Is there a way to set a list of words so that the program can recognize them?
I'm not really familiar with programing and such, so any help would be greatly appreciated! :D

This program has saved me so much time, thank you for providing it. Seriously amazing.

**Comment by Benko111** [comment]
That would actually be great. Some commercial transcription services like Speechmatics offer this for their implementation. A helper file user can edit before starting transcription process for Whisper to pick up words which are not common or very unusual, it works great.

**Comment by rheaster** [comment]
Not sure if there's been any progress on this front, but would love any help
